<launch>
  <machine name="hinotoritz" address="192.168.100.188" env-loader="/root/ros/devel/env.sh" user="hinotoritz" password="hinotoritz" />

  <node pkg = "rospkg-ogameasure"
        name = "GPDVC15_100"
        type = "GPDVC15_100.py"
        required="true"
        output="screen"
        machine="hinotoritz"
        >
        <param name="~host" value="192.168.100.44" />
        <param name="~gpibport_list" value="[4,5]" />

    <remap from = '/dev/gpdvc15_100rs/__IP__/port_4/i_cmd' to = '/dev/gpdvc15_100rs/ip_192_168_100_44/port_4/i_cmd'/>
    <remap from = '/dev/gpdvc15_100rs/__IP__/port_5/i_cmd' to = '/dev/gpdvc15_100rs/ip_192_168_100_44/port_5/i_cmd'/>

  </node>


</launch>
